go.mod

```diff
@@ -70,3 +70,5 @@ require (
 	sigs.k8s.io/yaml v1.2.0 // indirect
 	upper.io/db.v3 v3.7.1+incompatible
 )
+
+replace github.com/pingcap/tidb => github.com/lysu/tidb v1.1.0-beta.0.20210315025716-13b962a765d6
```

Comment:
https://github.com/pingcap/tidb/pull/23046 has been merged, the dependency can be updated now